project-base/app/AppKernel.php

```diff
@@ -48,6 +50,8 @@ public function registerBundles()
             new Shopsys\FrameworkBundle\ShopsysFrameworkBundle(),
             new Shopsys\ReadModelBundle\ShopsysReadModelBundle(), // has to be loaded after ShopsysFrameworkBundle because it overrides Twig `image` function
             new Shopsys\ShopBundle\ShopsysShopBundle(), // must be loaded as last, because translations must overwrite other bundles
+            new FOS\RestBundle\FOSRestBundle(),
```

Comment:
If it's not required to be loaded in some specific order, please sort it alphabetically, add a comment otherwise.